manifests/stage-whereabouts-cni/0050-whereabouts-ds.yaml

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 NVIDIA
```

Comment:
anything changed here ? (github marked this file as "changed since last review")

Reply:
added nodeAffinity, to prevent deploying on master